non-standard.json

```diff
@@ -14,6 +14,7 @@
   "org.scala-lang scalap":                           "version",
   "amplab spark-indexedrdd":                         "pom",
   "me.leaf akka-persistence-android":                "pom",
+  "com.typesafe config":                             "pom",
```

Comment:
http://search.maven.org/#artifactdetails%7Ccom.typesafe%7Cconfig%7C1.3.1%7Cbundle

It would be: `"com.typesafe config": "java",`. I did not implement this mode yet.

`pom` would read from the pom file and find a Scala version from the scala-library. In your case, you don't have a Scala target (Scala 2.10 transitively via lift-json).

https://github.com/scalacenter/scaladex/blob/master/data/src/main/scala/ch.epfl.scala.index.data/project/ProjectConvert.scala#L38